Documentation/ReleaseNotes/5.9.md

```diff
@@ -0,0 +1,17 @@
+# SwiftPM 5.9 Release Notes
+
+## Cross compilation
+
+SwiftPM now supports cross compilation based on the Swift SDK bundle format. While the feature is still considered experimental, we invite users to try it out and provide feedback.
+
+## Package registry
+
+SwiftPM can now publish to a registry following the specification defined in SE-0391, as well as support signed packages, which may be required by a registry. Trust-on-first-use (TOFU) validation checks can now use signing identities in addition to fingerprints, and are enforced for source archives as well as package manifests.
+
+## Other improvments
+
+The CompilerPluginSupport module enables defining macro targets. Macro targets allow authoring and distributing custom Swift macros as APIs in a library.
```

Comment:
Should the various code-related references such as specific APIs or modules use `code voice`?